src/devices/nm-device-bridge.c

```diff
@@ -170,17 +170,18 @@ typedef struct {
 	const char *sysname;
 	gboolean default_if_zero;
 	gboolean user_hz_compensate;
+	gboolean only_with_stp;
```

Comment:
Please make these types `bool`. Otherwise it wastes 9*8 bytes (on common 64 bit archs), for no benefit.

Reply:
ok